### PR TITLE
Sn/prop types/branding

### LINKED ISSUE #8: Add NYT branding to meet requirements
All applications must be accompanied by a Times API logo on any page or screen that displays Times API content or data. The logo must link directly to https://developer.nytimes.com.

More info: https://developer.nytimes.com/branding

### LINKED ISSUE #44: Add type checking with propTypes!!!

### PR DESCRIPTION
**Pull request by NakiNorton**
#### What's this PR do?
- Adds NYT API developer branding as determined in terms of use
- Adds type checking with propTypes
- Fixes/adds styling due to latest merge
- Fixes changes to search that were modified in one of the merges
- Enables upper/lower case characters to be used in search

Type of changes made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Styling- no new features

#### Where should the reviewer start?
- App.js 

#### Why is this change required? What problem does it solve?
- Fixes bugs coming out of merges and improves user experience

#### Were there any challenges that arose while implementing this feature? If so, how were they addressed?
- Ran into bugs because we had moved some methods/variable from Books to App

#### Any background context you want to provide?
- I went beyond the original scope of what I had intended to use this branch for

#### What are the relevant tickets?
- resolves #44 
- resolves #8

#### Screenshots (if appropriate)
-  n/a

#### Questions:
- n/a

### CODE REVIEW

File: src/Components/Nav/Nav.js [hunk 1]
```diff
@@ -13,10 +14,15 @@ const Nav = () => {
       <ul>
         <li><NavLink to='/' className='nav-link home-link' aria-label='navigate to home page'>HOME</NavLink></li>
         <li><NavLink to='/favorites' className='nav-link reading-list-link' aria-label='navigate to reading list'>READING LIST</NavLink></li>
+        <li><a href='https://developer.nytimes.com'> 
+          <img className='nyt-icon' alt='new york times api logo' src={nytIcon} /> 
+        </a></li>
```
Comment (kip-west) [inline src/Components/Nav/Nav.js:19]: Yay! Now we're not plagiarizing!

**Comment by amykarnaze** [comment]
Great fixes and additions

File: src/Components/Book/Book.css [hunk 2]
```diff
@@ -17,18 +17,24 @@
 
 .card-text {
   font-family: 'Amatic SC', cursive;
-  font-size: 26px;
-  align-self: center;
+  align-self: left;
```
Comment (NicoleLatifi) [inline src/Components/Book/Book.css:20]: praise: Great! I think this will be more legible now.

File: src/Components/Book/Book.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './Book.css'
 import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
```
Comment (NicoleLatifi) [inline src/Components/Book/Book.js:6]: Thank you for adding in the prop-types!

File: src/Components/ReadingListBook/ReadingListBook.js [hunk 4]
```diff
@@ -4,13 +4,17 @@ import { removeFavorite } from '../../actions'
...
 import { bindActionCreators } from 'redux';
 import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
 
 const ReadingListBook = ({ title, image, id, removeFavorite }) => {
 
   return (
     <article className='reading-list-book-card'>
-      <img className="card-image" alt={title} src={image} />
+      <Link to={`/${id}`}>
```
Comment (kip-west) [inline src/Components/ReadingListBook/ReadingListBook.js:13]: Great fix!! I like this solution, so that we don't add more components than we have to.

**Review by NicoleLatifi — APPROVED** [review]
Thanks for adding in the prop-types!

**Review by kip-west — APPROVED** [review]

**Pull request merged**